src/screens/ChannelListScreen/index.tsx

```diff
@@ -76,7 +77,7 @@ const ChannelListScreen = ({route}) => {
 
   return (
     <>
-      <StatusBar translucent={false} />
+      <StatusBar translucent={false} barStyle={'light-content'} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `barStyle` prop of the `StatusBar` component has been set to `'light-content'`. This will make the text and icons in the status bar light (typically white) which is suitable for dark backgrounds. Make sure this change aligns with your app's design and doesn't affect readability on different screens.
```diff
- <StatusBar translucent={false} />
+ <StatusBar translucent={false} barStyle={'light-content'} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->